### DISCUSSION TITLE
Welcome to spotfire-mods Discussions!

### DISCUSSION THREAD

**Discussion by objerke** [TIBCOSoftware/spotfire-mods#87] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

To share ideas please use the [TIBCO Ideas Portal](https://ideas.tibco.com/?category=6928343615694750134) and tag your Idea with `Mods`. Also use the Ideas Portal to vote for others' ideas.

Published mods can be found on the [TIBCO Exchange](https://community.tibco.com/exchange/tags/mods-43806/product/541).

**Comment by JGrotex** [comment]
Great to see this here on GH 👍

**Reply by mrkwpalmer** [reply]
Great to be here!

**Comment by hski-github** [comment]
Hey folks! Great to connect.

**Comment by jleviaguirre** [comment]
Hello everyone! I wonder how to create "channels" to discuss a particular topic. For example, if I want to share or discuss about the limitations of mod properties someone could come up with compressing the string data or requesting a binary mod property data type or even splitting the data across multiple mod properties. So, is there a way to create sections or everything is going to be here in one big    thread? I notice a "reference and issue, pull request or discussion" button on this text editor, but not sure how to use it.

**Reply by objerke** [reply]
Hi! Isn't there a "New discussion" button on the [Discussions](https://github.com/TIBCOSoftware/spotfire-mods/discussions) page? 
![image](https://user-images.githubusercontent.com/8665943/176842294-6b9e5b4e-2a66-49a9-9e73-51d4c633f235.png)

**Reply by hski-github** [reply]
Yes, there is (if you are logged in)

**Reply by heberleh** [reply]
There are the Categories in the Discussions, maybe those can be changed.
However, I believe that bug reports, feature request, enhancements... are topics to be discussed and managed in the Issues section, not Discussions, since they are connected to code and are made in the direction of improving the software.

**Reply by jleviaguirre** [reply]
Thanks guys for the advice.
